crates/uv/src/commands/project/init.rs

```diff
@@ -806,18 +811,69 @@ fn pyproject_project(
         dependencies = []
     "#,
         readme = if no_readme { "" } else { "\nreadme = \"README.md\"" },
-        authors = author.map_or_else(String::new, |author| format!("\nauthors = [\n    {} \n]", author.to_toml_string())),
+        authors = author.map_or_else(String::new, |author| format!("\nauthors = [\n    {}\n]", author.to_toml_string())),
         requires_python = requires_python.specifiers(),
     }
 }
 
 /// Generate the `[build-system]` section of a `pyproject.toml`.
-fn pyproject_build_system() -> &'static str {
-    indoc::indoc! {r#"
-        [build-system]
-        requires = ["hatchling"]
-        build-backend = "hatchling.build"
-    "#}
+/// Generate the `[tool.]` section of a `pyproject.toml` where applicable.
+fn pyproject_build_system(package: &PackageName, build_backend: ProjectBuildBackend) -> String {
+    let module_name = package.as_dist_info_name();
+    match build_backend {
```

Comment:
Do these build backend have forever stability guarantees (somewhere in their docs/github issues where we can link them?) or do we need to put version bounds on them?

Reply:
We currently don't put bounds on hatchling either, do we want to upper bound all of them?